### PR TITLE
Run mosquitto mqtt brokers in CI

### PR DESCRIPTION
**Pull request by anschweitzer**
Run two moquitto mqtt-broker containers in CI. In support of that: 
* Parameterize the port used to connect to each broker (currently read from settings.py). 
* Update the test_settings.py to use the same hardcoded port used in CI.

### CODE REVIEW

File: gw_spaceheat/actors/scada_base.py [hunk 1]
```diff
@@ -17,6 +17,7 @@ class ScadaBase(ActorBase):
     def __init__(self, node: ShNode, logging_on=False):
         super(ScadaBase, self).__init__(node=node, logging_on=logging_on)
         self.gwMqttBroker = settings.GW_MQTT_BROKER_ADDRESS
```
Comment (jessicamillar) [inline gw_spaceheat/actors/scada_base.py:19]: Should we refactor the other `settings` calls with the getattr pattern?

File: test/test_settings.py [hunk 2]
```diff
@@ -1,7 +1,9 @@
-LOCAL_MQTT_BROKER_ADDRESS = "broker.hivemq.com"
+LOCAL_MQTT_BROKER_ADDRESS = "localhost"
+LOCAL_MQTT_BROKER_PORT = 1883
 LOCAL_MQTT_USER_NAME = None
 
-GW_MQTT_BROKER_ADDRESS = "mqtt.eclipseprojects.io"
+GW_MQTT_BROKER_ADDRESS = "localhost"
+GW_MQTT_BROKER_PORT = 18831
```
Comment (jessicamillar) [inline test/test_settings.py:6]: can you update `template_settings` to have the `GW_MQTT_BROKER_ADDRESS` and `LOCAL_MQTT_BROKER_ADDRESS`  as well?

File: gw_spaceheat/actors/scada_base.py (hunk 1 above)
Comment (anschweitzer) [reply gw_spaceheat/actors/scada_base.py:19]: I think we should refactor settings to be an object you pass, by default read from env, but that’s means updating all users (you me and Pi) so I didn’t want to do it until we discus.

Comment (anschweitzer) [reply gw_spaceheat/actors/scada_base.py:19]: Yes.

File: test/test_settings.py (hunk 2 above)
Comment (anschweitzer) [reply test/test_settings.py:6]: Yes.

Comment (anschweitzer) [reply test/test_settings.py:6]: Done

**Review by jessicamillar — COMMENTED** [review]
👍

**Pull request merged**